activation/activation.go

```diff
@@ -713,8 +724,20 @@ func (b *Builder) broadcast(ctx context.Context, atx scale.Encodable) (int, erro
 }
 
 // getPositioningAtx returns atx id with the highest tick height.
-func (b *Builder) getPositioningAtx(ctx context.Context, nodeID types.NodeID) (types.ATXID, error) {
-	logger := b.log.With(log.ZShortStringer("smesherID", nodeID))
+// publish epoch is used for caching the positioning atx.
+func (b *Builder) getPositioningAtx(
+	ctx context.Context,
+	nodeID types.NodeID,
+	publish types.EpochID,
+) (types.ATXID, error) {
+	logger := b.log.With(log.ZShortStringer("smesherID", nodeID), zap.Uint32("publish epoch", publish.Uint32()))
+	b.posAtxFinder.finding.Lock()
+	defer b.posAtxFinder.finding.Unlock()
+	if found := b.posAtxFinder.found; found != nil && found.forPublish == publish {
+		logger.Info("using cached positioning atx", log.ZShortStringer("atx_id", found.id))
```

Comment:
I think this should be `Debug` instead of `Info`

Reply:
We are spamming logs too much. Printing this line for every identity besides the first adds no additional information in my oppinion.

Reply:
The problem with spamming logs is caused by logs repeated all the time. This log will be emitted N-1 times every two weeks so it's hardly spamming. Anyway, I can change it.